design-at-cornell/src/homepage/Homepage.tsx

```diff
@@ -1,10 +1,108 @@
 import React from 'react';
 import styles from './Homepage.module.css';
 import AppPlaceholderImage from '../placeholder/AppPlaceholderImage';
-
+import {
+  HomepageVerticalFlex,
+  SubComponentHorizontalFlex,
+  SubComponentVerticalFlex,
+} from '../components/ContainerStyles';
+import {
+  Title,
+  Subtitle,
+  SubComponent,
+  SubComponentTitle,
+  ExploreSubtitle,
+} from '../components/HomepageStyles';
+import design_courses from '../static/images/design_courses.svg';
+import majors_minors from '../static/images/majors_minors.svg';
+import orgs_icon from '../static/images/orgs_icon.svg';
+import events_icon from '../static/images/events_icon.svg';
+import resources_icon from '../static/images/resources_icon.svg';
+import community_icon from '../static/images/community_icon.svg';
 const Homepage: React.FC = () => (
   <div className={styles.Homepage} data-testid="Homepage">
     <AppPlaceholderImage />
+    <HomepageVerticalFlex>
+      <Title>Your ultimate resource for all things design at Cornell University</Title>
+      <Subtitle>
+        Connecting and inspiring designers, faculty, and current/future design students.
+      </Subtitle>
+    </HomepageVerticalFlex>
+
+    <SubComponentVerticalFlex>
+      <SubComponentHorizontalFlex>
+        <SubComponent onClick={() => window.open('/explore-courses')}>
```

Comment:
Since we are only redirecting to other pages within the d@c website, it makes more sense to do it in the same tab.